README.md

```diff
@@ -1 +1,19 @@
-Easypay Checkout SDK
+# Easypay Checkout SDK
+
+## To install dependencies:
+
+    `yarn install`
+
+## To build:
+
+    `yarn build`
+
+## To publish:
+
+### Public access:
+
+    `npm publish --access public`
+
+### Private access:
+
+    `npm publish`
```

Comment:
Is this true even though we have
```
  "publishConfig": {
    "access": "public"
  },
```
In `package.json`? It seems to me the default is public with that `package.json`.
And according to https://docs.npmjs.com/cli/v7/using-npm/config, it doesn't seem to change after initial publish (we would need to call `npm access`).

Maybe just keep `npm publish`?

Reply:
Yes if the config access is already setted we can only use `npm publish`.